projects/ngx-charts-on-fhir/src/lib/fhir-mappers/simple-observation-mapper.service.ts

```diff
@@ -64,12 +64,14 @@ export class SimpleObservationMapper implements Mapper<SimpleObservation> {
       },
       annotations: resource.referenceRange?.map<ChartAnnotation>((range) =>
         merge({}, this.annotationOptions, {
+          display: true,
```

Comment:
should we add `display: true` to the default ANNOTATION_OPTIONS?

Reply:
no @semicolin we don't need to display: true explicitly, so I have removed it.

Reply:
looks like it's still there. do you have more commits to push?

Reply:
I have updated PR, removed  `display: true` from simple-observation-mapper.service.ts